src/components/CommentCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+import { getCommets } from "../utils/utils";
+import "../styling/commentCard.css";
+
+
+function CommentCard({ reviewId }) {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    getCommets(reviewId).then((comments) => {
+      setComments(comments);
+    });
+  }, [reviewId]);
+
+  return <div>{comments.map((comment) => {
+    return <div className="comments" key= {comment.comment_id}>
+        <p>Author {comment.author}</p>
+        <p>{comment.body}</p>
+        <p>{comment.created_at}</p>
```

Comment:
this timestamp isn't user friendly, try manipulating the data to display it or trying a package that will convert it into a readable date for you, I like moment.js

Reply:
Done.